modules/microshift-restoring-backups.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// * microshift/microshift_backup_and_restore/microshift-auto-recover-manual-backup.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-restoring-backups_{context}"]
+= Restoring backups using the auto-recovery feature
+
+You can restore backups after system events that remove or damage required data.
+
+You can use the following procedure to restore backups.
```

Comment:
Not a huge deal but it might sound stronger if you removed "you can" from the beginning:
```suggestion
Use the following procedure to restore backups.
```

Reply:
done